src/plugins/auto/src/bind_multi_schedule.hpp

```diff
@@ -24,5 +24,9 @@ class BinderMultiSchedule : public AutoSchedule {
     void init(const ScheduleContext::Ptr& sContext) override;
     Pipeline GetPipeline(const IInferPtr& syncRequestImpl, WorkerInferRequest** WorkerInferRequest) override;
     virtual ~BinderMultiSchedule();
+
+private:
+    std::mutex _devInferMutex;
+    std::unordered_map<InferenceEngine::IInferRequestInternal::Ptr, WorkerInferRequest*> _devInfer;
```

Comment:
can we use snake style for the variables from now on for newly add codes?
I'm on api 2.0, and this will help further redundant re-work

Reply:
updated